electron/js/certutils.js

```diff
@@ -59,11 +59,9 @@ const pins = [
 ];
 
 module.exports = {
-  hostnameShouldBePinned (hostname) {
-    return pins.some(pin => pin.url.test(hostname.toLowerCase().trim()));
-  },
+  hostnameShouldBePinned: hostname => pins.some(pin => pin.url.test(hostname.toLowerCase().trim())),
```

Comment:
Is there a real benefit to this?

Reply:
the original version is more readable I would say..

Reply:
It's shorter :smiley: